packages/peregrine/lib/talons/CheckoutPage/PaymentInformation/usePaymentMethods.js

```diff
@@ -0,0 +1,25 @@
+import { useQuery } from '@apollo/react-hooks';
+import { useCartContext } from '../../../context/cart';
+
+export const usePaymentMethods = props => {
+    const { queries } = props;
+    const { getPaymentMethodsQuery } = queries;
+    const [{ cartId }] = useCartContext();
```

Comment:
I think we actually do need `useFieldState` since the "current selected method" is actually the "current checked radio" and not what is selected in the gql data/cart. So for now I've at least moved the use of the field state into the `usePaymentMethods` talon to reduce unnecessary prop threading.

Reply:
Good change. 👍